apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -35,9 +205,217 @@ export default class ProjectCard extends React.Component {
       isPublicGallery && isDetailView && projectData.publishedAt;
     const noTimeOnCardStyle = shouldShowPublicDetails ? {} : styles.noTime;
 
+    const {
+      showReportAbuse,
+      isAnyCheckboxSelected,
+      checkboxes,
+      showSubmittedHeader,
+      submitButtonEnabled,
+      reportButtonHovered,
+      reportButtonPressed,
+      showSubmitConfirmation,
+    } = this.state;
+
+    const captchaSiteKey = getScriptData('projects').recaptchaSiteKey;
+
+    const script = document.createElement('script');
+    script.src = 'https://www.google.com/recaptcha/api.js';
+    script.id = 'captcha';
+    script.async = true;
+    script.defer = true;
+    window.onCaptchaSubmit = token => this.onCaptchaVerification(token);
+    window.onCaptchaExpired = () => this.onCaptchaExpiration();
+    script.onload = () => this.setState({loadedCaptcha: true});
+    document.body.appendChild(script);
+
     return (
       <div className="project_card">
+        {showReportAbuse ? (
+          <AccessibleDialog
+            onClose={this.cancel}
+            className={style.reportAbusePopUp}
+          >
+            {showSubmitConfirmation ? (
+              <div
+                style={{
```

Comment:
To Denys' point, I think at least moving all of this inline styling into the `styles` object below would help with readability.